automation/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/notebooks/NotebookBioconductorKernelSpec.scala

```diff
@@ -0,0 +1,299 @@
+/**
+  *  1. Check if singleCellExp, DESeq2 etc %in% installed.packages()
+  *  2. BiocManager::install() for
+  *  3. BiocManager::version() == 3.6.1
+  *  4. BiocManager::install('rsbml')
+  *  5. BiocManager::install('RCurl') ## tests libcurl
+  *  6. "XML" ##libxml
+  *  7. "graphviz" RGraphviz
+  *  8. "BiocSklearn"
+  *  9. "rhhdf5"
+  *  10. openbabel - ChemmineOB
+  *  11. gsl - DirichletMultinomial
+  *  12. gtk - RGtk2
+  *  13. magick++ EBImage
+  *  14. protobuf- protolite
+  *  15. databse stuff - RMySQL
+  *  16. jags - rjags
+  */
+
+package org.broadinstitute.dsde.workbench.leonardo.notebooks
+
+import org.broadinstitute.dsde.workbench.leonardo.{ClusterFixtureSpec, LeonardoConfig}
+import org.broadinstitute.dsde.workbench.service.util.Tags
+import org.scalatest.DoNotDiscover
+
+import scala.concurrent.duration._
+import scala.language.postfixOps
+
+/**
+  * This spec verifies notebook functionality specifically around the R-Bioconductor kernel.
+  */
+@DoNotDiscover
+class NotebookBioconductorKernelSpec extends ClusterFixtureSpec with NotebookTestUtils {
+  override val jupyterDockerImage: Option[String] = Some(LeonardoConfig.Leonardo.biocImageUrl)
+  "NotebookBioconductorKernelSpec" - {
+
+    "should use Bioconductor version 3.9" in { clusterFixture =>
+
```

Comment:
cluster creation takes at least 3 min currently...I think it's better to use one cluster...Something like https://github.com/DataBiosphere/leonardo/blob/develop/automation/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/notebooks/NotebookHailSpec.scala#L21 (mmm, ignore the naming in the test :) )...We do loose some nice things compared to the way you're doing now tho.

Reply:
nvm..think I misread how `ClusterFixtureSpec`..seems cluster is only created once before all tests in the same file.